src/core/utils/remote-feature-config.ts

```diff
@@ -29,22 +29,27 @@ export const getRemoteConfigFeatures = async () => {
         [RemoteFeature.TC_VERSION]: undefined
     });
 
-    await firebase.config().fetch(duration);
-    await firebase.config().activateFetched();
-    const objects = await firebase
-        .config()
-        .getValues([
-            RemoteFeature.NEAR,
-            RemoteFeature.COSMOS,
-            RemoteFeature.DEV_TOOLS,
-            RemoteFeature.TC_VERSION
-        ]);
+    try {
+        await firebase.config().fetch(duration);
+        await firebase.config().activateFetched();
 
-    featuresConfig = {};
-    // Retrieve values
-    Object.keys(objects).forEach(key => {
-        featuresConfig[key] = objects[key].val();
-    });
+        const objects = await firebase
+            .config()
+            .getValues([
+                RemoteFeature.NEAR,
+                RemoteFeature.COSMOS,
+                RemoteFeature.DEV_TOOLS,
+                RemoteFeature.TC_VERSION
+            ]);
+
+        featuresConfig = {};
+        // Retrieve values
+        Object.keys(objects).forEach(key => {
+            featuresConfig[key] = objects[key].val();
+        });
+    } catch (err) {
+        //
```

Comment:
i think we need to handle this, featuresConfig will be undefined, we should set some defaults